vsintegration/src/FSharp.ProjectSystem.FSharp/ProjectFlagsService.fs

```diff
@@ -0,0 +1,123 @@
+﻿// Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.VisualStudio.FSharp.ProjectSystem
+
+open System
+open System.ComponentModel.Composition
+open System.Collections.Generic
+open Microsoft.VisualStudio.FSharp.LanguageService
+open Microsoft.VisualStudio
+open Microsoft.VisualStudio.Shell
+open Microsoft.VisualStudio.Shell.Interop
+open Microsoft.Build.Execution
+
+(*
+[<Export(typeof<ProjectFlagsService>)>]
+type internal ProjectFlagsService 
+    [<ImportingConstructor>]
+    (
+        [<Import(typeof<SVsBuildManagerAccessor>)>] accessor : IVsBuildManagerAccessor
```

Comment:
I experimented a bit and it turns out exports from other assemblies like FSharp.Editor work fine and are resolved with IComponentModel. Anything declared here fails.